templates/digital-asset-template/frontend/graphql/indexer.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLClient } from "graphql-request";
+import { getSdk } from "./generated/indexer/sdk";
+import { TokenQueryQuery } from "./generated/indexer/operations";
+
+export const INDEXER_GRAPHQL_ENDPOINT =
+  process.env.INDEXER_GRAPHQL_ENDPOINT ??
+  "https://indexer-testnet.staging.gcp.aptosdev.com/v1/graphql";
+
+const indexerClient = new GraphQLClient(INDEXER_GRAPHQL_ENDPOINT);
+export const indexer = getSdk(indexerClient);
```

Comment:
I think we can simply use SDK queries or at least aptos.indexerQuery - we dont really need the graphql codegen

Reply:
Thank you for the reference!